SoftLayer/managers/ordering.py

```diff
@@ -322,7 +322,7 @@ def get_preset_by_key(self, package_keyname, preset_keyname, mask=None):
 
         return presets[0]
 
-    def get_price_id_list(self, package_keyname, item_keynames):
+    def get_price_id_list(self, package_keyname, item_keynames, core):
```

Comment:
This function should still be usable if there is no core count, so have the default value be `core=None`

Reply:
Done